Tourniquet duplicity and stringtable fixes

YetheSamartaka opened the pull request:
- removes settable tourniquet time option since it is added by ACE

YetheSamartaka committed:
Tourniquet duplicity

- removes settable tourniuqet time option since it is added by ACE

Conrimaceogain commented:
LGTM, let's make sure that things play happily with the ACE settings and then we should be good to merge.

YetheSamartaka commented:
Indeed

YetheSamartaka committed:
various stringtable fixes

-fixed pulse-oximeter log for French language
-minor fixes for Turkish language
- minor adjustments for Czech language

Tomcat-SG commented on addons/breathing/stringtable.xml:
```diff
@@ -153,8 +153,8 @@
-            <French>Oxymètre de pouls: HR: 1% SpO2: 2%</French>
-            <Turkish>Nabız Oksimetresi: HR:% 1 Sp02:% 2</Turkish>
+            <French>Oxymètre de pouls: HR: %1 SpO2: %2</French>
+            <Turkish>Nabız Oksimetresi: HR:%1 Sp02:%2</Turkish>
```
```suggestion
            <Turkish>Nabız Oksimetresi: HR: %1 Sp02: %2</Turkish>
```

Tomcat-SG commented on addons/breathing/stringtable.xml:
```diff
@@ -215,9 +215,9 @@
-				% 1Sp02% 70'in üzerinde -> Evet
-				% 1Sp02% 70'in altında -> Hayır -> eşek. Nefes</Turkish>
+				%1yeterince havalandırıldı mı?
+				%1Sp02% 70'in üzerinde -> Evet
```
```suggestion
				%1Sp02 70% 'in üzerinde -> Evet
```

Tomcat-SG replied:
```suggestion
				%1Sp02 70% 'in üzerinde -> Evet
```

Tomcat-SG commented on addons/breathing/stringtable.xml:
```diff
@@ -215,9 +215,9 @@
-				% 1Sp02% 70'in altında -> Hayır -> eşek. Nefes</Turkish>
+				%1yeterince havalandırıldı mı?
+				%1Sp02% 70'in üzerinde -> Evet
+				%1Sp02% 70'in altında -> Hayır -> eşek. Nefes</Turkish>
```
```suggestion
				%1Sp02 70% 'in altında -> Hayır -> eşek. Nefes</Turkish>
```

Tomcat-SG reviewed (changes requested):
Pls fix these things. Im not sure if the % on the turkish translation could cause at this position an error, because it is followed by a number, but it is better in this way, so it cant cause any errors

YetheSamartaka committed:
Turkish stringtable fix

Turkish stringtable fix

Co-authored-by: Tomcat <51529018+Tomcat-SG@users.noreply.github.com>
